apps/E2E/src/Chip/pages/ChipPageObject.ts

```diff
@@ -0,0 +1,35 @@
+import { BasePage, By } from '../../common/BasePage';
+import { AndroidAttribute } from '../../common/consts';
+import { CHIP_TESTPAGE, CHIP_TEST_COMPONENT, CHIP_TEXT, HOMEPAGE_CHIP_BUTTON } from '../consts';
+
+class ChipPageObject extends BasePage {
+  /******************************************************************/
+  /**************** UI Element Interaction Methods ******************/
+  /******************************************************************/
+
+  async verifyTextContent(text: string): Promise<boolean> {
```

Comment:
This function is not needed. Please use compareAttribute() from BasePage

Reply:
Thanks for catching, replaced function with compare attribute